lib/bundle/brew_installer.rb

```diff
@@ -2,6 +2,7 @@ module Bundle
   class BrewInstaller
     def self.install(name, options = {})
       new(name, options).install_or_upgrade
+      BrewServices.restart(name) if options[:restart_service]
```

Comment:
i don't think we can do that, because if we change if to postfix and remove `else`. What will happened if `result` is `true` but `restart_service` is `false`, the method will return `false`, which should be `true`